### PR TITLE
fix(rln): Remove resources folder, update missed docs

### PR DESCRIPTION
**Pull request by seemenkina**
Part of #237 

- Remove parameter `resources_folder` from config
- Update comments and docs which were missed in #245

### CODE REVIEW

File: rln/README.md [hunk 1]
```diff
@@ -14,18 +15,19 @@ cd zerokit/rln
...
+
 ```bash
 cargo make build
 cargo make test
 ```
 
 ### Compile ZK circuits
 
-The `rln` (https://github.com/privacy-scaling-explorations/rln) repository, which contains the RLN circuit implementation is a submodule of zerokit RLN. 
+The `rln` (https://github.com/privacy-scaling-explorations/rln) repository, which contains the RLN circuit implementation is a submodule of zerokit RLN.
```
Comment (rymnc) [inline rln/README.md:26]: this should be changed to 
```suggestion
The `rln` (https://github.com/rate-limiting-nullifier/circom-rln) repository, which contains the RLN circuit implementation is a submodule of zerokit RLN.
```

File: rln/src/public_api_tests.rs [hunk 2]
```diff
@@ -543,8 +535,7 @@ fn test_rln_proof() {
     }
 
     // We create a new RLN instance
-    let input_buffer =
-        Cursor::new(json!({ "resources_folder": TEST_RESOURCES_FOLDER }).to_string());
+    let input_buffer = Cursor::new(json!({}).to_string());
```
Comment (rymnc) [inline rln/src/public_api_tests.rs:538]: maybe this can be abstracted away `generate_input_buf` for example, and reused in all tests

**Review by rymnc — COMMENTED** [review]
LGTM, thanks
added nit inline

File: rln/src/public_api_tests.rs [hunk 3]
```diff
@@ -588,11 +579,10 @@ fn test_rln_proof() {
     rln.generate_rln_proof(&mut input_buffer, &mut output_buffer)
         .unwrap();
 
-    // output_data is [ proof<128> | share_y<32> | nullifier<32> | root<32> | epoch<32> | share_x<32> | rln_identifier<32> ]
-    let mut proof_data = output_buffer.into_inner();
```
Comment (seemenkina) [inline rln/src/public_api_tests.rs:592]: find missed

File: rln/README.md [hunk 4]
```diff
@@ -82,14 +83,14 @@ use rln::protocol::*;
...
 // - the tree height;
-// - the circuit resource folder (requires a trailing "/").
+// - the tree config, if it is not defined, the default value will be set
 let tree_height = 20;
-let resources = Cursor::new("../zerokit/rln/resources/tree_height_20/");
+let input = Cursor::new(json!({}).to_string());
 
 // We create a new RLN instance
-let mut rln = RLN::new(tree_height, resources);
+let mut rln = RLN::new(tree_height, input);
```
Comment (rymnc) [inline rln/README.md:93]: future pr, I don't think we need to pass in the tree height as an argument anymore, we're fixing it to 20.

**Review by rymnc — APPROVED** [review]
LGTM

File: rln/README.md (hunk 4 above)
Comment (seemenkina) [reply rln/README.md:93]: Should it be added as an issue in this issue?

**Pull request merged**